src/components/data/EntitiesContent.js

```diff
@@ -359,6 +359,87 @@ const EntitiesContent = ({
     ])])
   }
 
+  const entitiesSelected = !_.isEmpty(selectedEntities)
+  const canEdit = !Utils.editWorkspaceError(workspace)
+
+  const renderEditMenu = () => {
+    return !snapshotName && h(MenuTrigger, {
+      side: 'bottom',
+      closeOnClick: true,
+      content: h(Fragment, [
+        h(MenuButton, {
+          onClick: () => setDeletingEntities(true)
+        }, 'Delete')
+      ])
+    }, [h(ButtonSecondary, {
+      disabled: !canEdit || !entitiesSelected,
+      tooltip: Utils.cond(
+        [!canEdit, () => 'You do not have permission to modify this workspace'],
+        [!entitiesSelected, () => 'Select rows to edit'],
+        () => 'Edit selected data'
+      ),
+      style: { marginRight: '1.5rem' }
+    }, [icon('edit', { style: { marginRight: '0.5rem' } }), 'Edit'])])
+  }
+
+  const renderExportMenu = ({ columnSettings }) => {
+    const isSet = _.endsWith('_set', entityKey)
+    const isSetSet = entityKey.endsWith('_set_set')
```

Comment:
If I'm guessing the meaning of `isSetSet` correctly, consider

```suggestion
    const isSetOfSets = entityKey.endsWith('_set_set')
```

Similarly in `Data.js`.

Reply:
Yes, that's definitely clearer.

Reply:
Done in [c1d3d05](https://github.com/DataBiosphere/terra-ui/pull/2907/commits/c1d3d050c4f5f8de04e5e814935496904c51c856).